packages/std/src/errors/std_error.rs

```diff
@@ -480,7 +481,8 @@ pub enum CheckedMultiplyFractionError {
     #[error("{0}")]
     Overflow(#[from] OverflowError),
 }
-
+/// Represents errors during safe multiplication of a number by a fraction.
```

Comment:
This one now says "fraction", but is in fact not used with the `Fraction` type. It is returned by [this function](https://github.com/costa2400/cosmwasm/blob/04933f00247a0bff6930d5e2281e3fe50065e1cf/packages/std/src/math/int128.rs#L108) (and its friends for the different integer sizes).
The whole operation is really more about multiplying and dividing at once to avoid overflowing in-between.